functions.php

```diff
@@ -0,0 +1,161 @@
+<?php
+require_once 'dbConn.php';
+
+/*
+ * Is used to select data from the database and return it as a multidimensional array, designed specifically for the about me content
+ *
+ * @param - PDO - $db - this is where the $db var should be used otherwise the function can't access the database(scope)
```

Comment:
no hyphens in docblocks, while it may be easier to read, your IDE wont pick them up and is not industry standard